### PR TITLE
Document reverse DNS setup recommendations

### LINKED ISSUE #1252: DNS documentation misses reverse DNS setup
https://mailu.io/master/dns.html should contain an explanation about reverse DNS and why it's important.

### PR DESCRIPTION
**Pull request by micw**
## What type of PR?

documentation

## What does this PR do?

Add docs about setting up reverse DNS

### Related issue(s)
- closes #1252

## Prerequistes
none

### CODE REVIEW

**Comment by micw** [comment]
bors try

**Comment by micw** [comment]
bors try

File: docs/dns.rst [hunk 1]
```diff
@@ -49,6 +49,23 @@ And for another domain, ``myotherdomain.com`` for example:
...
+
+You can verify this with
+
+.. code-block:: bash
+
+  nslookup a.b.c.d
+
+Reverse DNS must be set up by the "owner" of the IP address which is usually your hosting provider. You can look it up with ``whois a.b.c.d`` in most cases.
+
+With incorrect reverse DNS setup, most mail systems will reject you emails as spam.
```
Comment (hoellen) [inline docs/dns.rst]:
```suggestion
With incorrect reverse DNS setup, most mail systems will reject your emails as spam.
```
Just a small typo.

**Pull request merged**